pom.xml

```diff
@@ -96,7 +96,6 @@
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
-            <scope>test</scope>
```

Comment:
@prondzyn this is not a good idea. Would be great to stay away from extra dependencies. Can we do that?